Makefile

```diff
@@ -128,7 +128,7 @@ ifeq ($(TRAVIS),true)
 test: run-tests-within-container
 else
 ifeq ($(CHECKSTYLE),0)
-test: test-with-database
+test: test-with-database test-containers-compose
```

Comment:
It does not make sense to only call this when skipping checkstyle

Reply:
Modified, I think now is the correct place

Reply:
No, sorry. It also does not make sense to only call this when checkstyle is included. have you seen my comment in https://github.com/os-autoinst/openQA/pull/3773#pullrequestreview-607477161 ? Please read carefully that I mentioned there *two* lines. However this will still not be clean. This is why there is also https://github.com/os-autoinst/openQA/pull/3772/files which cdywan has invited you to review two days ago. I suggest we finish https://github.com/os-autoinst/openQA/pull/3772 first and then extend here.